resources/assets/css/_general.css

```diff
@@ -74,72 +46,37 @@ h3,
 
 h4,
 .header-4 {
-    @apply text-lg mb-1;
+    @apply text-lg mb-1 md:text-2xl;
 }
 
-/* Header Responsiveness */
-@screen md {
-    h1,
-    .header-1 {
-        @apply text-4xl mb-2;
+@layer base {
+    a:focus,
+    button:focus,
+    textarea:focus,
+    input:focus {
+        @apply outline-none;
     }
 
-    h2,
-    .header-2 {
-        @apply text-3xl;
+    button,
+    img {
+        @apply select-none;
+        -webkit-user-drag: none;
     }
 
-    h3,
-    .header-3 {
-        @apply text-2xl;
+    body {
+        @apply font-sans text-theme-secondary-700 antialiased;
     }
 }
 
-h1,
-.header-1,
-h2,
-.header-2,
-h3,
-.header-3,
-h4,
-.header-4 {
-    @apply leading-tight;
-}
-
-a:focus,
-button:focus,
-textarea:focus,
-input:focus {
-    @apply outline-none;
-}
-
-button,
-img {
-    @apply select-none;
-    -webkit-user-drag: none;
-}
-
-body {
-    @apply font-sans text-theme-secondary-700 antialiased;
-}
-
-.clear-transform {
-    transform: none;
-}
-
-.rotate-90 {
-    transform: rotate(90deg);
-}
-
-.rotate-180 {
-    transform: rotate(180deg);
-}
+@layer utilities {
```

Comment:
pushed `clear-transform` and `rotate-minus-90` to `utilities` layer. though they may not be even used in any of the projects, i wouldn't remove them here now